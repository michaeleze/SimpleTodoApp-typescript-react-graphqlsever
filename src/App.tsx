import React from 'react';
const Layout = React.lazy(() => import('./layout'));

const App = () => (
  <React.Suspense fallback={<div>Loading Page...</div>}>
    <Layout/>
  </React.Suspense>
  )

export default React.memo(App);
