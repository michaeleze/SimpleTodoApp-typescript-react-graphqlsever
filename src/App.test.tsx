import React from 'react';
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import App from './App';

it('should renders snapshot', () => {
  const {container} = render(<App />)
  expect(container).toMatchSnapshot();
});
