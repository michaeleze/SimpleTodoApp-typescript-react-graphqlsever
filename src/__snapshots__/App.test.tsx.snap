// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders snapshot 1`] = `
<div>
  <div>
    Loading Page...
  </div>
</div>
`;
